plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckAccuracyRadiusAnimator.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.maps.plugin.locationcomponent.animators
+
+import android.graphics.Color
+import androidx.annotation.ColorInt
+import com.mapbox.maps.plugin.locationcomponent.OnIndicatorAccuracyRadiusChangedListener
+
+internal class PuckAccuracyRadiusAnimator(private val accuracyRadiusChangedListener: OnIndicatorAccuracyRadiusChangedListener) :
+  PuckAnimator<Double>(Evaluators.DOUBLE) {
+
+  var enabled = false
```

Comment:
?
```suggestion
  private/internal var enabled = false
```

Reply:
It's an internal class, is this really needed?

Reply:
this may change in the future.
better to limit visibility as much as we can